ui/components/app/edit-gas-display/edit-gas-display.stories.js

```diff
@@ -1,34 +1,167 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Provider } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { action } from '@storybook/addon-actions';
+import configureStore from '../../../store/store';
+import testData from '../../../../.storybook/test-data';
+import { decGWEIToHexWEI } from '../../../helpers/utils/conversions.util';
+import { useGasFeeInputs } from '../../../hooks/gasFeeInput/useGasFeeInputs';
+import { EDIT_GAS_MODES, GAS_LIMITS } from '../../../../shared/constants/gas';
 import EditGasDisplay from '.';
 
 export default {
   title: 'Edit Gas Display',
```

Comment:
Follow new story sorting format
```suggestion
  title: 'Components/App/Edit Gas Display',
```